src/keycloak/Keycloak.Seeding/appsettings.json

```
@@ -0,0 +1,38 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "Serilog": {
+    "Using":  [ "Serilog.Sinks.Console" ],
+    "MinimumLevel": "Information",
```

The minimumLevel section should be changed into the following:

```suggestion
"MinimumLevel": {
    "Default": "Information",
    "Override": {
    "Microsoft": "Warning",
      "Microsoft.Hosting.Lifetime": "Information"
    }
},
```